app/controllers/miq_policy_controller/alerts.rb

```diff
@@ -583,14 +583,16 @@ def alert_valid_record?(alert)
       unless value_greater_than && is_integer?(value_greater_than)
         add_flash(_(template_error % [">", non, an_integer]), :error)
       end
-      unless value_greater_than.to_i.between?(0, 100)
```

Comment:
So, key was that this sentence
value_greater_than.to_i.between?(0, 100)
works different for empty/nil values than
value_greater_than = value_greater_than.to_i
unless value_greater_than.between?(0, 100)

In a first view it looked pretty close to me.